app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(
+        data: list,
+        wife_key: str = "wife",
+        husband_key: str = "husband"
+) -> list:
+    person_list = [Person(person_data["name"],
+                   person_data["age"]) for person_data in data]
```

Comment:
It is not a good practice to place two arguments that are used in an instance initialization on a new line in this way, place them on a same line, e.g.:
```suggestion
    person_list = [Person(person_data["name"], person_data["age"]) 
                   for person_data in data]
```